conf/application.conf

```diff
@@ -58,12 +72,6 @@ play.http.router = prod.Routes
 controllers {
   # 300 is the default, you may need to change this according to your needs
   confidenceLevel = 300
```

Comment:
this config can be removed since we don't use `auth` (also the `microservice.services.auth`

Reply:
REmoved